test/Removal.test.ts

```diff
@@ -51,7 +52,16 @@ describe('Removal', () => {
         formatTokenAmount(balance)
       );
       const expectedMarketSupply = 1000;
-      const tokenIds = [10, 11, 12, 13];
+      const { supplier } = hre.namedAccounts;
```

Comment:
This block of code changed to prevent issues with supplier addresses with value 0x0. Everything else is formatting noise

Reply:
mmm yeah, this one's gotten me before